src/pymor/bindings/scipy.py

```diff
@@ -175,13 +175,16 @@ def apply_inverse(op, V, options=None, least_squares=False, check_finite=True,
     |VectorArray| of the solution vectors.
     """
 
-    assert isinstance(op, NumpyMatrixOperator)
-    # TODO Use to_matrix(op) after sparse format issue has been resolved.
     assert V in op.range
 
+    if isinstance(op, NumpyMatrixOperator):
+        matrix = op._matrix
+    else:
+        from pymor.algorithms.to_matrix import to_matrix
+        matrix = to_matrix(op)
```

Comment:
I guess the if branch is not necessary, since `to_matrix` should cover that case, but I don't have strong opinions about it.

Reply:
This is intended as a shortcut for computations with reduced models where a few less function calls might (or might not ...) payoff in terms of performance.

Reply:
Ok :+1: